Typescript error when importing jest-extended

**Bug**

jest-extended version: 1.0.0

.ts file:
```ts
import * as matchers from "jest-extended";
```

I get error:
File '.../node_modules/jest-extended/types/index.d.ts' is not a module.

Also, after upgrading to 1.0.0 unassigned import does not work any more:
```ts
import "jest-extended";
```

So:
Unassigned import stoped working in version 1.0.0
And wildcard import does not work in typescript because of invalid .d.ts file.

I tried every option here and on the TS setup in the docs with v1.0.0 and I also get `foo.test.ts:95:20 - error TS2339: Property 'toIncludeAllPartialMembers' does not exist on type 'JestMatchers...`

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-interface CustomMatchers<R> {
+interface CustomMatchers<R> extends Record<string, any> {
   /**
    * Note: Currently unimplemented
    * Passing assertion

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -884,3 +884,8 @@ declare namespace Vi {
   // eslint-disable-next-line @typescript-eslint/no-empty-interface
   interface AsymmetricMatchersContaining extends CustomMatchers<any> {}
 }
+
+declare module 'jest-extended' {
+  const matchers: CustomMatchers<any>;
+  export = matchers;
+}

